internal/graph/graph_test.go

```diff
@@ -663,42 +315,17 @@ func TestConnectedObjectGraph_RelationshipIngresss(t *testing.T) {
 		},
 		{
 			name: "Ingresses for non-assignable relation",
-			model: &openfgapb.AuthorizationModel{
-				TypeDefinitions: []*openfgapb.TypeDefinition{
-					{
-						Type: "organization",
-						Relations: map[string]*openfgapb.Userset{
-							"viewer":   typesystem.This(),
-							"can_view": typesystem.ComputedUserset("viewer"),
-						},
-						Metadata: &openfgapb.Metadata{
-							Relations: map[string]*openfgapb.RelationMetadata{
-								"viewer": {
-									DirectlyRelatedUserTypes: []*openfgapb.RelationReference{
-										typesystem.DirectRelationReference("organization", ""),
-									},
-								},
-							},
-						},
-					},
-					{
-						Type: "document",
-						Relations: map[string]*openfgapb.Userset{
-							"parent": typesystem.This(),
-							"view":   typesystem.TupleToUserset("parent", "can_view"),
-						},
-						Metadata: &openfgapb.Metadata{
-							Relations: map[string]*openfgapb.RelationMetadata{
-								"parent": {
-									DirectlyRelatedUserTypes: []*openfgapb.RelationReference{
-										typesystem.DirectRelationReference("organization", "viewer"),
-									},
-								},
-							},
-						},
-					},
-				},
-			},
+			model: `
+			type organization
```

Comment:
@craigpastro this test was actually bad before because `document#parent` was restricted to `organization#viewer` (which is an invalid type restriction for a tupleset relation).

I modified the model to make more sense, but I noticed it fails now.

Given the model
```
type organization
  relations
    define viewer: [organization]
    define can_view as viewer

type document
  relations
    define parent: [organization]
    define view as can_view from parent
```

I would expect `RelationshipIngresses(document#view, organization)` to return `["organization#viewer" (direct)]`

So the assertion is correct, but right now we're getting the response `["organization#viewer" (direct), "document#view" (direct)]`. The second in the list is erroneous. 

Mind taking a look into this?

Reply:
Will open a follow up PR with more tests and will fix the issues in that PR.